test/e2e/lib/browser-manager.ts

```diff
@@ -6,8 +6,9 @@
 /**
  * External dependencies
  */
-import { Browser, BrowserContext, chromium } from 'playwright';
+import { chromium } from 'playwright';
 import config from 'config';
+import type { Browser, BrowserContext } from 'playwright';
```

Comment:
This addresses https://github.com/Automattic/wp-calypso/pull/51146#discussion_r601741698.